### PR TITLE
Update to sportbukkit 1.11

### PR DESCRIPTION
**Pull request by Pablete1234**

### CODE REVIEW

File: src/main/java/in/twizmwaz/cardinal/module/modules/damageIndicator/PlayerBoundingBox.java [hunk 1]
```diff
@@ -146,23 +145,17 @@ public void sendSpawnPackets(Player viewer) {
...
-                        loc.getX() + (x * DamageIndicator.OFFSET), loc.getY() , // X, Y
-                        loc.getZ() + (z * DamageIndicator.OFFSET),              // and Z coords
-                        0, 0, 0,                                                // X, Y and Z Motion
-                        (byte)2, (byte)0, (byte)2,                              // Yaw, Pitch and Head Pitch
-                        dataItems);                                             // Metadata
+                        zombieID.get(i++), UUID.randomUUID(), 54,             // Entity id, UUID, and type (Zombie)
+                        loc.getX() + (x * DamageIndicator.OFFSET), loc.getY(),// X, Y
+                        loc.getZ() + (z * DamageIndicator.OFFSET),            // and Z coords
+                        0, 0, 0,                                              // X, Y and Z Motion
+                        (byte)2, (byte)0, (byte)2,                            // Yaw, Pitch and Head Pitch
```
Comment (twizmwazin) [inline src/main/java/in/twizmwaz/cardinal/module/modules/damageIndicator/PlayerBoundingBox.java]: spaces

File: src/main/java/in/twizmwaz/cardinal/module/modules/damageIndicator/PlayerBoundingBox.java [hunk 2]
```diff
@@ -146,23 +145,17 @@ public void sendSpawnPackets(Player viewer) {
...
         for (int x = -1; x < 2; x += 2) {
             for (int z = -1; z < 2; z += 2) {
                 Packet spawnPacket = new PacketPlayOutSpawnEntityLiving(
-                        zombieID.get(i), UUID.randomUUID(), 54,                 // Entity id, UUID, and type (Zombie)
-                        loc.getX() + (x * DamageIndicator.OFFSET), loc.getY() , // X, Y
-                        loc.getZ() + (z * DamageIndicator.OFFSET),              // and Z coords
-                        0, 0, 0,                                                // X, Y and Z Motion
-                        (byte)2, (byte)0, (byte)2,                              // Yaw, Pitch and Head Pitch
-                        dataItems);                                             // Metadata
+                        zombieID.get(i++), UUID.randomUUID(), 54,             // Entity id, UUID, and type (Zombie)
```
Comment (twizmwazin) [inline src/main/java/in/twizmwaz/cardinal/module/modules/damageIndicator/PlayerBoundingBox.java]: Just to make sure you know, the function will be passed the original value of i, and increment i after. To pass the incremented value, ++i

Comment (Pablete1234) [inline src/main/java/in/twizmwaz/cardinal/module/modules/damageIndicator/PlayerBoundingBox.java]: intended, the old behaviour was passing "i" and running "i++" after that block of code

**Comment by twizmwazin** [comment]
merged

**Pull request closed without merging**